.env

```diff
@@ -38,3 +38,5 @@ RABBITMQ_PORT=5672
 
 # LDAP USER MAPPING
 LDAP_USER_GROUP=
+
+NEXT_PUBLIC_SYSTEM_VERSION=latest
```

Comment:
Why not simply `SYSTEM_VERSION`? This is how we define it in ARTEMIS.

Reply:
If you add the NEXT_PUBLIC_ prefix, you can use the env var on the client side code. Its a nextjs trick

Reply:
A OK, did not know this. Resolving this comment.